interpreter/interpreter.py

```diff
@@ -688,6 +688,11 @@ def respond(self):
               traceback.print_exc()
             error = traceback.format_exc()
             time.sleep(3)
+         except openai.error.InvalidRequestError as ie:
+            msg = str(ie)
+            print(f"Error from OpenAI: {msg}")
+            print("This could mean: Your API Key could be invalid, or anything else blocks openinterpreter to access GPT4. Please check your OpenAI Account.")
+
```

Comment:
I think this needs to move to [`interpreter/terminal_interface/validate_llm_settings.py`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/terminal_interface/validate_llm_settings.py) now that the codebase has been refactored.